apps/dashboard/src/components/workflow-editor/control-input/control-input.tsx

```diff
@@ -145,6 +152,7 @@ export function ControlInput({
         }}
         highlightedVariableKey={highlightedVariableKey}
         enableTranslations={shouldEnableTranslations}
+        translationValueInput={ControlInput}
```

Comment:
workflow and layouts editors use different inputs that have the related context, so they can show the variable suggestions